components/layout/display_list_builder.rs

```diff
@@ -2781,6 +2793,13 @@ fn position_to_offset(position: LengthOrPercentage, Au(total_length): Au) -> f32
     }
 }
 
+fn outset_to_length(outset: LengthOrNumber, border_width: Au) -> Au {
```

Comment:
Let's move this to `model.rs` and call this `specified_outset`, for symmetry with other properties like border-radius.

Let's also make the doc comment of that reference https://drafts.csswg.org/css-backgrounds-3/#border-image-outset.